src/main/java/com/cloudant/client/org/lightcouch/internal/URIBuilder.java

```diff
@@ -106,6 +127,11 @@ public URIBuilder path(String path) {
         return this;
     }
 
+    public URIBuilder userInfo(String userInfo) {
+        this.userInfo += userInfo;
```

Comment:
I temporarily need a way to set the username and password into the URI.  If not, the proper authentication will not be added in HttpConnection's execute.